app/todos/resources.py

```diff
@@ -0,0 +1,85 @@
+# Ask about getting branches up to date once committed
+
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, Todo
+from app.lib.response_util import buildOkResponse
+
+
+def _returnTodo(todo):
+    """Private method to convert a todo object into a dictionary.
+
+    Args:
+        todo - A todo model.
+    """
+    return {
+            "id": todo.id,
+            "subject": todo.subject
```

Comment:
Do you think we should return more information than this? Imagine that you are the browser and want to communicate to display to user a todo item. What information will you need?